include/wx/msw/listctrl.h

```diff
@@ -216,6 +216,12 @@ class WXDLLIMPEXP_CORE wxListCtrl: public wxListCtrlBase
     void SetItemFont( long item, const wxFont &f);
     wxFont GetItemFont( long item ) const;
 
+    // Checkbox state of an item
+    bool GetCheckboxesEnabled() const;
```

Comment:
I think it would be more consistent to call this `AreCheckboxesEnabled()` or maybe `HasCheckboxes()`.